gp-templates/translation-set-edit.php

```diff
@@ -4,6 +4,10 @@
 	gp_project_links_from_root( $project ),
 	gp_link_get( $url, $locale->english_name . 'default' != $set->slug? ' '.$set->name : '' ),
 ) );
+
+// jQuery is required for the 'translation-set-form' template.
+gp_enqueue_scripts( array( 'jquery' ) );
```

Comment:
Is there a reason why you're using gp_enqueue_script*s*() and not gp_enqueue_script()?